assert/assertions_test.go

```diff
@@ -1217,6 +1217,7 @@ func Test_containsElement(t *testing.T) {
 
 	list1 := []string{"Foo", "Bar"}
 	list2 := []int{1, 2}
+	list3 := []byte(`Hello World Hello`)
```

Comment:
Let be consistent here and below:
```suggestion
	list3 := []byte("Hello World Hello")
```